features/search_dependents.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Pages depending on search
+
+  @normal
+  Scenario Outline: check results for pages relying on search
+    Given I am testing through the full stack
+    And I force a varnish cache miss
+    When I visit "<Path>"
+    Then I should see search results at css "<Search results selector>"
+
+    Examples:
```

Comment:
I'm wondering if these selectors are too brittle (eg it's very likely that `app-c-topic-list` will change soon). Aren't most of these things tested in their own smokey tests?